parsing/src/processor.ts

```diff
@@ -6,8 +6,8 @@ import * as iconv from 'iconv-lite';
 import {Recipe} from "./interfaces/Recipe";
 import {Part,PartDataInterface} from "./interfaces/Part";
 import {getItems,fixItemNames,fixTurbofuel} from './parts';
-import {getRecipes} from './recipes';
-import {getProducingBuildings, getPowerConsumptionForBuildings} from './buildings';
+import {getProductionRecipes,getPowerGeneratingRecipes} from './recipes';
```

Comment:
```suggestion
import {getProductionRecipes, getPowerGeneratingRecipes} from './recipes';
```

Reply:
Why is this one getting a space, but the other interface imports are not? Shouldn't they all have a space after the comma or not?